team/bundles/org.eclipse.compare.win32/src/org/eclipse/compare/internal/win32/WordComparison.java

```diff
@@ -198,10 +200,20 @@ public void createWorkingCopy(String baseDocument, String revisedDocument, Strin
 		resetSite(null);
 		OleAutomation application = createApplication();
 		try {
-			OleAutomation document = openDocument(application, revisedDocument);
+			final String primaryDocument;
```

Comment:
Note: While I could just switch the 2 variables here, I think it is an anti-pattern to modify references to method arguments (Kotlin even won't let you do so), so I better introduced new variables here.